### ISSUE TITLE
Waiting for NES mini... when trying to add games.

### ISSUE THREAD

**Issue by mghtyred** [ClusterM/hakchi2#415]
I've been trying to get a solution for this all day, and nothing. I follow all the instructions, then when it's time to add the roms, I get that same message. I have no idea what I'm doing wrong here, or even if I am doing something wrong. Please help.

**Comment by ghost** [comment]
Go down to your Taskbar and select the "Safely Remove Hardware and Eject Media" icon like you are trying to safely disconnect a USB device. If you see "Eject CLV-S-NESY" as an option then your NESC is connected properly to your PC.

Look for the storage bar at the bottom of the Hakchi2 window. Before you select games to add, it should show how much of your Classic's storage is currently being used in green. If you don't see any green, Hakchi2 is not recognizing that your Classic is connected to your PC.

I've had the same issue myself. I'm not sure what the problem was. I was using the USB cable that came with the console, and tried both a direct USB connection and a powered USB hub. What ended up working for me was checking the connection of the USB cord in the back of the Classic (putting slight pressure on it). On mine, moving it slightly caused my PC to play the device disconnect and device connect tones. After a couple times, Hakchi2 eventually recognized my Classic was connected (the green in the storage bar showed up). Then I was able to flash kernels.

**Comment by mghtyred** [comment]
I have not seen "Eject CLV-S-NESY" yet... but I may not have checked every time. I will experiment with this. Yes, it is always green, and moving. There is also a red dot at the lower left hand side of the Hakchi2 window. 

I've also heard trying a much earlier version of Hakchi2 may do it. May try that today as well.

**Comment by angelbless** [comment]
Not working for me either. Storage is fine. I can install/unistall modules just fine...and it detects Nes Mini on the Devices too.

**Comment by mghtyred** [comment]
UPDATE:

 When I connect via holding down RESET then power, I do see "EJECT NES MINI"  

I just tried using an earlier version of Hakchi with no success. I am beginning to worry that this version of NES Classic has anti-hacking measures in place.

**Comment by mghtyred** [comment]
Tried starting all over with Windows Defender off because someone in another forum suggested this might be causing a conflict. No luck there either. I'm at a loss.

**Comment by mghtyred** [comment]
As expected, I think there is a bigger problem at work here. I just tried to use what should have been a feature of the new Kernel and held down down arrow and SELECT to reset the console, and nothing. 

I don't think the kernel is actually flashing. What could be causing this?

**Comment by mghtyred** [comment]
SOLVED: 

 Needed to attach to a USB 2.0 port. I thought I had tried this already, but apparently had just gone from one 3.0 port to another 3.0 port. NESClassic does not like 3.0.

**Comment by angelbless** [comment]
@ @maskedmaniac acutally, messing with the back of the Nes connector helped. Dont know why but thxs!

**Comment by nealcrocker** [comment]
I have been using 2.17c with this issue. I am using Windows 10 Pro. For some reason it can't seem to find the NES Mini. I downgraded to version 213 before the upgrade to USB library changed to LibWinUsb and then it began to detect the NES MIni. The next problem was that the LED light remained when trying to add game with new kernel. It seemed to work, however when I connected the Mini to the TV it appeared as nothing was updated at all and the original kernel was still there and the default games. Any help would be appreciated. I have yet to try 2.17d, but somehow I don't think what is in the release notes can assist here. I am in Australia and I have the following model # BFC-CLV-S-NESB-AUS-C1. Has anyone else had this problem with Australian versions.
